tests/Mutant/MetricsCalculatorTest.php

```diff
@@ -41,42 +41,40 @@ public function test_it_shows_zero_values_by_default(): void
         $this->assertSame(0.0, $calculator->getCoveredCodeMutationScoreIndicator());
     }
 
+    private function addMutantProcess(MetricsCalculator $calculator, int $resultCode, int $count = 1): void
```

Comment:
Should we move this below the test method?

Reply:
It is in reading order and in close proximity. I don't think it would be any better if it would be below.